js/src/wp-seo-admin-global.js

```diff
@@ -128,3 +128,47 @@
 	window.wpseoMakeDismissible = wpseoMakeDismissible;
 	window.wpseoDismissLink = wpseoDismissLink;
 }());
+
+(function( $ ) {
```

Comment:
Because all files are now browserified we don't need this pattern anymore. You can just do:

```js
var $ = jQuery;
```